src/Driver.php

```diff
@@ -64,10 +64,11 @@ public function ack(string $queueName, Envelope $envelope);
      * @param   string $queueName The queue from whcih the message came
      * @param   $envelope The message envelope -- should be the same instance
      *          returned from `dequeue`
+     * @param   int $delay The number of seconds to wait before retrying
      * @throws  Exception\DriverError when something goes wrong
      * @return  Envelope The new envelope for the retried job.
      */
-    public function retry(string $queueName, Envelope $envelope) : Envelope;
+    public function retry(string $queueName, Envelope $envelope, int $delay = 0) : Envelope;
```

Comment:
No spaces around the `=` here please: `int $delay=0`

Reply:
How come you don't use PSR-2 style?